locale/en_US/manager.po

```diff
@@ -536,7 +536,7 @@ msgid "manager.statistics.region"
 msgstr "Region"
 
 msgid "manager.statistics.reports"
-msgstr "Report Generator"
+msgstr "Reports"
```

Comment:
In the new `.po` format, when a locale key changes in the `en_US` translation, we can mark it "fuzzy" in all other translations. Translators will then be alerted that it has changed in the weblate translation tools.

To mark it fuzzy in other languages, you would add a small comment above the `msgid`, like this:

```
#, fuzzy
msgid "..."
msgstr "..."
```